include/deal.II/base/trilinos_utilities.h

```diff
@@ -180,6 +184,32 @@ namespace Utilities
     duplicate_map(const Epetra_BlockMap &map, const Epetra_Comm &comm);
   } // namespace Trilinos
 #endif
+
+
+#ifdef DEAL_II_TRILINOS_WITH_TPETRA
+  namespace Trilinos
```

Comment:
We already have `namespace TrilinosWrappers`. Why not put it in there? If you did, you don't have to qualify the use of this function at all if you're inside that namespace (which should be all the places where you're using it) and in that case you can play the following game:
```
namespace TrilinosWrappers
{
#if DEAL_II_TRILINOS_VERSION_LT(too old)
   ...provide make_rcp...
#  else
    using Teuchos::make_rcp;
#endif


...some code inside namespace TrilinosWrappers:
x = make_rcp<...>(...);
```
The latter code uses `TrilinosWrappers::make_rcp()` if provided (namely, if Trilinos is too old) and falls back to `Teuchos::make_rcp` otherwise. Once we get to a point where we *require* a new enough Trilinos library, we can just remove the `#if` part.

Reply:
Sure, we can move that into the `namespace TrilinosWrappers`. But then the question arises: in which file should I place that function?

With that argument, moving that function into the `namespace TpetraWrappers` could be even more convenient since I am using that function quite heavily in that namespace.

Reply:
Ah, I had not realized that `TpetraWrappers` are not in namespace `TrilinosWrappers` :-(

I guess then this is ok as is.